include/nbl/ext/MitsubaLoader/SContext.h

```diff
@@ -110,8 +111,8 @@ namespace MitsubaLoader
 			return key;
 		}
 
-		static auto emissionProfileSamplerParams(const CElementEmissionProfile& profile) {
-			asset::ISampler::SParams res = { asset::ISampler::ETC_CLAMP_TO_EDGE, asset::ISampler::ETC_CLAMP_TO_EDGE, asset::ISampler::ETC_CLAMP_TO_EDGE, asset::ISampler::ETBC_INT_OPAQUE_BLACK, asset::ISampler::ETF_NEAREST, asset::ISampler::ETF_NEAREST, asset::ISampler::ESMM_NEAREST, 0u, false, asset::ECO_ALWAYS };
+		static auto emissionProfileSamplerParams(const CElementEmissionProfile& profile, const asset::CIESProfileMetadata& meta) {
+			asset::ISampler::SParams res = { asset::ISampler::ETC_CLAMP_TO_EDGE, asset::ISampler::ETC_CLAMP_TO_EDGE, asset::ISampler::ETC_CLAMP_TO_EDGE, asset::ISampler::ETBC_INT_OPAQUE_BLACK, asset::ISampler::ETF_LINEAR, asset::ISampler::ETF_LINEAR, asset::ISampler::ETF_LINEAR, 0u, false, asset::ECO_ALWAYS };
```

Comment:
if you want the octahedral mapping to work right, you need REPEAT edge clamping on U and V